vkubelet/vkubelet.go

```diff
@@ -91,6 +92,11 @@ func New(nodeName, operatingSystem, namespace, kubeConfig, taint, provider, prov
 		if err != nil {
 			return nil, err
 		}
+	case "mock":
+		p, err = mock.NewMockProvider(nodeName, operatingSystem, daemonEndpointPort)
```

Comment:
Add the `internalIP`, and return it in the NodeAddresses()

Reply:
:+1: